store/store_test.js

```diff
@@ -27,58 +25,35 @@ authPage.fillRegistrationEmailField(Date.now() + '@test.com');
 authPage.clickCreateAccount();
 createAccountPage.fillNewAccountFields(user);
 createAccountPage.clickRegister();
-I.see('MY ACCOUNT');*/
+I.see('MY ACCOUNT');
+});*/
 
 
 Scenario('buy product', async ({ I, homePage, authPage, myAccountPage, productPage, cartPage }) => {
     homePage.clickSignIn();
     authPage.login();
-    myAccountPage.myAccountField();
+    myAccountPage.getMyAccountField();
     I.amOnPage('http://automationpractice.com/index.php?id_product=2&controller=product');
     let productPrice = await productPage.getProductPrice();
     console.log(productPrice);
-    let numberProductPrice = productPrice.slice(1);
     productPage.clickAddToCart();
     let shippingPrice = await cartPage.getShippingPrice();
     console.log(shippingPrice);
-    let numberShippingPrice = shippingPrice.slice(1);
-    numberProductPrice = Number(numberProductPrice);
-    numberShippingPrice = Number(numberShippingPrice);
-    let numberTotalPrice = (numberProductPrice + numberShippingPrice);
-    numberTotalPrice = Number(numberTotalPrice);
+    numberTotalPrice = (productPrice + shippingPrice);
```

Comment:
`let` is missing
you can move `productPrice + shippingPrice` to `assertEqual()` to avoid creating variable for this easy instruction